### PR TITLE
DOCS-1494: Redesign the archive procedure

### PR DESCRIPTION
**Pull request by girishshankaran**
Product Version(s):

Issue:
(https://tigera.atlassian.net/browse/DOCS-1494)

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Comment by girishshankaran** [comment]
@ctauchen Can you review this one?

**Comment by girishshankaran** [comment]
3.16 change is the latest one.

File: calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx [hunk 1]
```diff
@@ -6,17 +6,9 @@ description: Install an older patch release of Calico Enterprise.
...
-
-Install an older patch release of {{prodname}}.
-
-## Concepts
-
-Installing a patch release is easy:
-
-1. Download and "un-tar" a patch release to a local directory.
-1. Apply the downloaded manifests.
+You can install a previous patch release in certain scenarios, such as when the latest patch is unstable or for other business-related reasons.
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx]: A suggestion, to use as you like:

```
Installing the most recent patch version of {{prodname}} {{version}} ensures that you have the latest bug fixes and improvements.
But you can always install, or downgrade to, a previous patch version.
Installing an older patch can help if you encounter problems with the latest patch, or if you're required to install only a specific patch version.
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx [hunk 2]
```diff
@@ -27,9 +19,9 @@ This feature is:
 
 ## How to
 
-1. Go to the [Releases tab](https://docs.tigera.io/releases), and navigate to the appropriate release (3.0 and later).
+1. Download your patch release from the following table.
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx]:
```suggestion
1. Download the release archive for your patch version from the following table:
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx [hunk 3]
```diff
@@ -27,9 +19,9 @@ This feature is:
...
 
-1. In the left navigation, click [Release notes](../release-notes/index.mdx), and click the link, **Release archive**.
+   <ReleaseArchiveTable/>
 
 1. Untar the **release-x.x.x.tgz** to a local directory.
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx]: I notice that the format for the file name does not match the current format.

File: calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx (hunk 1 above)
Comment (girishshankaran) [reply calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx]: Done

File: calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx (hunk 2 above)
Comment (girishshankaran) [reply calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx]: Done

File: calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx (hunk 3 above)
Comment (girishshankaran) [reply calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx]: Changed it to `release-x.x.x-y.y.y.tgz`

Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx]: Maybe let's go with `release-vx.y.z-vx.y.z.tgz`. Does that seem less confusing to you?

**Pull request merged**